### PR TITLE
VATRP-3041 Android: LinphoneCoreException Fix double quote issue when…

### PR DESCRIPTION
**Pull request by patrickmwatson**
Fix double quote issue when receiving strings in autoconfig throwing LinphoneCoreException

### CODE REVIEW

File: src/org/linphone/vtcsecure/Utils.java [hunk 1]
```diff
@@ -14,6 +14,26 @@
  */
 public class Utils {
 
+    public static String removeExtraQuotesFromStringIfPresent(String inputStr){
+        String result = inputStr;
+        //Check if string has extra quotes
+        if(inputStr.startsWith("\"")&&inputStr.endsWith("\"")){
+            result=removeQuotesFromStartAndEndOfString(inputStr);
+        }
+        return result;
```
Comment (zackm0571) [inline src/org/linphone/vtcsecure/Utils.java:23]: Add sanity check to return empty String if inputStr is null.

File: src/org/linphone/vtcsecure/Utils.java [hunk 2]
```diff
@@ -14,6 +14,26 @@
...
+
+    private static String removeQuotesFromStartAndEndOfString(String inputStr) {
+        String result = inputStr;
+        int firstQuote = inputStr.indexOf('\"');
+        int lastQuote = result.lastIndexOf('\"');
+        int strLength = inputStr.length();
+        if (firstQuote == 0 && lastQuote == strLength - 1) {
+            result = result.substring(1, strLength - 1);
+        }
+        return result;
```
Comment (zackm0571) [inline src/org/linphone/vtcsecure/Utils.java:35]: Add sanity check to return empty String if inputStr is null.

**Comment by shareefalis** [comment]
+1

**Comment by zackm0571** [comment]
+1

**Comment by shareefalis** [comment]
Even better!

**Pull request merged**